litmus-portal/frontend/src/pages/WorkflowDetails/index.tsx

```diff
@@ -1,16 +1,21 @@
+/* eslint-disable no-unused-expressions */
```

Comment:
Is this required @Jonsy13  ?

Reply:
yes @amityt, it was required for the RR score calculation. With the new API endpoint, it will be removed completely.